thesis-fe/src/components/container/AddDestination.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react';
+import styled from 'react-emotion'
+import { fontFamily } from '../../helpers/constants';
+import { WizardMode } from './WizardMode';
+import { List } from './List';
+
+
+export class AddDestination extends Component {
+  state = {
+    input: '',
+    dictator: false,
+  }
+
+  componentDidMount() {
```

Comment:
> But I'm curious why don't you simply use `this.props.destination.chosenOne` instead of writing into the state?

To be honest, I had the dictator state value before I had suggestions or chosenOne properties in my props so I guess I didn't realise! I will refactor my components now